### PR TITLE
Adding a 'printing' notification for user

### PR DESCRIPTION
**Pull request by pgiraud**
Fixes partially #642.

### CODE REVIEW

File: geoportailv3/static/js/print/printdirective.js [hunk 1]
```diff
@@ -110,6 +110,11 @@ app.PrintController = function($scope, $timeout, ngeoCreatePrint,
   this['title'] = '';
 
   /**
+   * @type {boolean|undefined}
+   */
+  this['printing'] = undefined;
```
Comment (elemoine) [inline geoportailv3/static/js/print/printdirective.js]: Why not false?

File: geoportailv3/static/js/print/printdirective.js [hunk 2]
```diff
@@ -292,6 +298,7 @@ app.PrintController.prototype.handleGetStatusSuccess_ = function(ref, resp) {
   if (done) {
     // The report is ready. Open it by changing the window location.
     window.location.href = this.print_.getReportUrl(ref);
+    this['printing'] = false;
```
Comment (elemoine) [inline geoportailv3/static/js/print/printdirective.js]: Shouldn't you also reset `printing` to `false` in case of errors?

Comment (pgiraud) [inline geoportailv3/static/js/print/printdirective.js]: Indeed. Done.

**Comment by pgiraud** [comment]
@elemoine Do you think we can merge it now?

**Comment by elemoine** [comment]
We should! Thanks.

**Pull request merged**